test/e2e_node/inode_eviction_test.go

```diff
@@ -250,6 +253,12 @@ func runEvictionTest(f *framework.Framework, testCondition string, podTestSpecs
 			return nil
 		}, postTestConditionMonitoringPeriod, evictionPollInterval).Should(BeNil())
 
+		By("deleting evicted pods")
```

Comment:
The purpose of starting a new pod is not really to test eviction, but rather to make sure that other subsequent tests are not affected by this test.  This way, if there is anything wrong after the test (that we didnt already catch), we can link other test failures to the eviction test failure here.